### ISSUE TITLE
conflict with syntax-highlighting plug

### ISSUE THREAD

**Issue by day0n** [zsh-users/zsh-autosuggestions#673]
### To Reproduce

```sh
% zsh -df
% source path/to/zsh-autosuggestions.zsh
% ... # what do you do to reproduce?
```

### Desktop
 - OS + distribution: 
 - Zsh version: 
 - Plugin version:

**Comment by scrubmx** [comment]
I have the same issue:

* macOS Monterrey 12.4
* iTerm2 3.4.15
* zsh 5.8.1
* oh-my-zsh
* Plugin versions:
	* zsh-autosuggestions: v0.6.4  
	* zsh-syntax-highlighting: 0.8.0-alpha2-dev

* Both plugins active `plugins=(zsh-autosuggestions zsh-syntax-highlighting)`
<img width="388" alt="Screen Shot 2022-05-24 at 21 01 02" src="https://user-images.githubusercontent.com/2574295/170163520-7ffb5581-bdb4-4e90-9625-1f7ad361c988.png">

* Only autosuggestions active `plugins=(zsh-autosuggestions)`
<img width="392" alt="Screen Shot 2022-05-24 at 21 02 28" src="https://user-images.githubusercontent.com/2574295/170163652-ca7d7f19-5397-46b4-8714-3611ad09bb0c.png">

I'm also setting the color in my `.zshrc` file:
```sh
ZSH_AUTOSUGGEST_HIGHLIGHT_STYLE="fg=242"
```

**Comment by jimmyn** [comment]
I have the same issue

**Comment by jirutka** [comment]
Me too.

**Comment by pawel-omniaz** [comment]
Same here :/

**Comment by DimitriGilbert** [comment]
I have an identical problem,
One thing though, if I connect to a server through ssh, highlight appears properly...

**Comment by ihatemicrosoft-was-already-taken** [comment]
I have the very same behaviour as described by @DimitriGilbert 
However, I observed this only after updating to Ubuntu 22.04. I wonder if there's a connection.

**Comment by ihatemicrosoft-was-already-taken** [comment]
I was able to fix the issue by updating both autosuggestions and syntax-highlighting to their latest versions from GitHub (just cloned the repos).

**Comment by pawel-omniaz** [comment]
Yep, I can confirm - updating both plugins fixed it for me too on Mac OS :) Thanks! I thought those plugins are updated automatically by Oh My Zsh or something ;)

**Comment by DimitriGilbert** [comment]
same thing, and just like @pawel-omniaz , I didn't even think I was in charge of the update.

**Comment by shotexa** [comment]
Had the same issue, removed plugins from `~/.oh-my-zsh/custom/plugins/` and then re-installed them via

Syntax highlighting
```
git clone https://github.com/zsh-users/zsh-syntax-highlighting.git ${ZSH_CUSTOM:-~/.oh-my-zsh/custom}/plugins/zsh-syntax-highlighting
```

Autosuggestions
```
git clone https://github.com/zsh-users/zsh-autosuggestions ${ZSH_CUSTOM:-~/.oh-my-zsh/custom}/plugins/zsh-autosuggestions
```

**Comment by findtravishere** [comment]
this bug was driving me crazy and i couldnt fix it properly until i followed @shotexa 's instructions above. 
my main issue was i had no idea where my old plugins were being installed to so after reinstalling them it was still pointing to the old ones that weren't patched. 

just in case others somehow have it installed in another directory similar to mine even when running the default `git clone` commands, this was my path i did `rm -rf` and to activate the source for zsh-syntax-highlighting with.

`source /Users/CHANGE_THIS_WITH_YOUR_USER/.oh-my-zsh/custom/plugins/zsh-syntax-highlighting/zsh-syntax-highlighting.zsh`

**Comment by jirutka** [comment]
This problem persists on Zsh 5.8.1 even with the latest revision of both plugins but is no longer present on Zsh 5.9.
